packages/server/src/api/builds.js

```diff
@@ -64,7 +64,7 @@ export type BuildPostOptions = {
 };
 
 export type BuildPostCallbacks = {
-  onBuildInserted?: (comparator: BuildComparator) => Promise<Object>
+  onBuildInserted?: (comparator: BuildComparator) => Promise<string>
```

Comment:
I think this is a valid case to use `any` as opposed to `Object` or `string`. Let the developer decide what they want to return here.